src/server/scripts/Commands/cs_misc.cpp

```diff
@@ -1386,7 +1386,31 @@ class misc_commandscript : public CommandScript
 
         if (!target || !target->IsConnected())
         {
-            return false;
+            if (handler->HasLowerSecurity(nullptr, target->GetGUID()))
+                return false;
+
+            ObjectGuid::LowType guid = sCharacterCache->GetCharacterGuidByName(target->GetName()).GetCounter();
+
+            CharacterDatabasePreparedStatement* stmt = CharacterDatabase.GetPreparedStatement(CHAR_SEL_CHARACTER_HOMEBIND);
+            stmt->SetData(0, guid);
+
+            PreparedQueryResult result = CharacterDatabase.Query(stmt);
+
+            if (result)
+            {
+                uint32 mapId = (*result)[0].Get<uint32>();
+                uint32 zoneId = (*result)[1].Get<uint32>();
+                float posX = (*result)[2].Get<float>();
+                float posY = (*result)[3].Get<float>();
+                float posZ = (*result)[4].Get<float>();
+                float posO = (*result)[5].Get<float>();
+
+                Player::SavePositionInDB(mapId, posX, posY, posZ, posO, zoneId, target->GetGUID());
```

Comment:
```suggestion
                    Field* fieldsDB = result->Fetch();
                    WorldLocation loc(fieldsDB[0].Get<uint16>(), fieldsDB[2].Get<float>(), fieldsDB[3].Get<float>(), fieldsDB[4].Get<float>(), 0.0f);
                    uint32 zoneId = fieldsDB[1].Get<uint16>();

                    Player::SavePositionInDB(loc, zoneId, target->GetGUID(), nullptr);
```

from the .tele name $home command

Reply:
Ok, I'll check it out. Thank you.